src/utils/download.js

```diff
@@ -1,3 +1,4 @@
 export const downloadResources = (resources) => {
+  // eslint-disable-next-line no-console
   console.log('Download', resources);
 };
```

Comment:
This file is an artifact of a recursive copy from girder-vue, it can go away completely, but we can do that on another branch if you want.